src/drivers/uart/pl011.rs

```diff
@@ -0,0 +1,109 @@
+#![no_std]
+
+use register::register_bitfields;
+use register::mmio::{ReadOnly, ReadWrite};
+
+trait Driver {
```

Comment:
one thing I don't understand: could this driver trait be common to all our devices in oreboot? No need to do that now, just wondering.

This is cool.

Reply:
Yeah, it definitely should be. I just didn't know which directory to put it in.